doc/sphinx/source/examples/Tutorials/Two_Way_Wave_Equation/Two_Way_Wave_Equation.md

```diff
@@ -0,0 +1,219 @@
+# Tutorial: Comparing Finite Difference and Mimetic Difference Operators  
+### Solving the 1D Two-Way Wave Equation
+
+This tutorial demonstrates how to solve the **1D two-way wave equation** using both **standard finite differences (FD)** and **mimetic finite differences (MD)**.  
+We will use three main functions:
+
+1. [`finite_diff_two_way_wave_eq`](#finite_diff_two_way_wave_eq)
+2. [`mimetic_diff_two_way_wave_eq`](#mimetic_diff_two_way_wave_eq)
+3. [`comparison_two_way_wave_md_vs_fd`](#comparison_two_way_wave_md_vs_fd)
+
+The goal is to understand the **differences in accuracy, stability, and numerical properties** between traditional and mimetic operators. The two numerical functions have been created to be as similar as possible, so that only the changes necessary for mimetic difference operators are visible.
```

Comment:
```suggestion
The goal is to understand the **differences in accuracy, stability, and numerical properties** between traditional finite and mimetic difference operators. The two numerical functions have been created to be as similar as possible, so that only the changes necessary for mimetic difference operators are visible.
```